source/elements/oneDAL/source/terminology.rst

```diff
@@ -0,0 +1,64 @@
+.. _terminology:
+
+============
+Terminology
+============
+
+@table-begin:
+   column-sizes: [ 25% ]
+Term | Definition \\
+
+Batch Mode | The computation mode for an algorithm in oneDAL, where all the data needed for
+computation are available at the start and fits the memory of the device where computations will
+be performed. \\
+
+Dataset | A pair of a matrix $X$ of size $N \times p$ and a vector $Y$ of size $N$, where $X$
+contains $N$ feature vectors in rows and $Y$ stores all corresponding responses. Vector $Y$ is
+optional and may not be present if responses are unknown. \\
+
+DPC++ | DPC++ --- Technology (programming language, compiler and runtime) that
+is based on SYCL specification and extends it. \\
+
+Feature | A single column of matrix $X$ from a dataset. \\
+Feature vector | A single real-valued vector of length $p$ that encodes information about a real
+object or an event. \\
+
+GEMM | General Matrix Multiplication --- matrix multiplication algorithms in the following form $$C
+= \alpha A^\dagger B^\dagger + \beta C, \; \dagger \in \{1, T\}.$$ \\
+Gen9 | Codename of Intel GPU microarchitecture was used for performance analysis of the algorithms
```

Comment:
Do we need it at all here?
Generally, Intel specifics should not be in the vendor neutral specs

Reply:
removed